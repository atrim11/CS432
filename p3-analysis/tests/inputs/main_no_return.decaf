def void main() {
    void b;
    int c;
    return;
}