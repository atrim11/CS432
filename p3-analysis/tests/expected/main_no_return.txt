Void variable 'b' on line 2
'main' must return an integer
